active/0000-alternate-dependencies.md

```diff
@@ -0,0 +1,92 @@
+- Start Date: (2015-07-12)
+- RFC PR: (leave this empty)
+- ember-cli Issue: (leave this empty)
+
+# Summary
+
+Allow ember-cli to target another package.json file when checking its dependency list. 
```

Comment:
I like the idea of having alternative package.json files, might be nice for the ember-try project

cc @kategengler @mike-north